components/post_view/post_list/post_list.jsx

```diff
@@ -12,14 +12,14 @@ import EventTypes from 'utils/event_types.jsx';
 import GlobalEventEmitter from 'utils/global_event_emitter.jsx';
 import * as UserAgent from 'utils/user_agent.jsx';
 import * as Utils from 'utils/utils.jsx';
-import LoadingScreen from 'components/loading_screen.jsx';
-import DateSeparator from 'components/post_view/date_separator.jsx';
 
-import FloatingTimestamp from './floating_timestamp.jsx';
-import NewMessageIndicator from './new_message_indicator.jsx';
-import Post from './post';
-import ScrollToBottomArrows from './scroll_to_bottom_arrows.jsx';
-import CreateChannelIntroMessage from './channel_intro_message';
+import CreateChannelIntroMessage from 'components/post_view/channel_intro_message';
+import DateSeparator from 'components/post_view/date_separator.jsx';
+import FloatingTimestamp from 'components/post_view/floating_timestamp.jsx';
+import LoadingScreen from 'components/loading_screen.jsx';
+import NewMessageIndicator from 'components/post_view/new_message_indicator.jsx';
+import Post from 'components/post_view/post';
+import ScrollToBottomArrows from 'components/post_view/scroll_to_bottom_arrows.jsx';
 
```

Comment:
Move PostList to its own folder since it's mixed with other common/shared components.